### ISSUE TITLE
Rust dependencies not synced between Cargo and Bazel

### ISSUE THREAD

**Issue by daviddrysdale** [project-oak/oak#875]
Cargo puts the dependency version information into the `Cargo.lock` file, and `cargo-raze` stores its versions of dependencies in the generated `cargo/crates.bzl` file (and the `cargo/remote/*.BUILD` files).  However, these are not kept in sync; running `cargo-raze` seems to ignore what's in `Cargo.lock` (and indeed what's generated by `cargo metadata`) and picks its own choice of versions.

**Comment by daviddrysdale** [comment]
No longer relevant after #1016.
